templates/barbican/config/barbican-api-config.json

```diff
@@ -78,14 +78,16 @@
       "optional": true,
       "merge": true
     },
+    {{- if (index . "P11ClientDataPath") }}
     {
-      "source": "/var/lib/config-data/default/Chrystoki.conf",
-      "dest": "/usr/local/luna/Chrystoki.conf",
+      "source": "/etc/p11-client/*",
```

Comment:
should be `/var/lib/config-data/hsm`, right ?

Reply:
Done